.travis.yml

```diff
@@ -17,16 +15,10 @@ script:
   - npm run build:rollup
   - npm --prefix=server test
 
-before_install:
-  - npm install -g npm@6.9.0
-  - node --version
```

Comment:
Will the setup script still output, in the log, the npm and node versions used?

Reply:
Node will be the version set by the .nvmrc, so I didn't print it again. 

It would be worthwhile to print the npm version.